### PR TITLE
Added BFF

### PR DESCRIPTION
**Pull request by NikG77**
Product-service service API endpoint URL https://v36qxqri4i.execute-api.eu-west-1.amazonaws.com/dev
Cart-service API endpoint URL  http://nikg77-cart-api-dev.eu-west-1.elasticbeanstalk.com/api/profile
Bff-service service URL  http://nikg77-bff-api-develop.eu-west-1.elasticbeanstalk.com

5 - The bff-service should be deployed with Elastic Beanstalk. The bff-service call should be redirected to the appropriate service : product-service or CART. The response from the bff-service should be the same as if product-service or CART services were called directly

Call product-service via bff-service service URL (get products list)
GET http://nikg77-bff-api-develop.eu-west-1.elasticbeanstalk.com/products
Call cart-service via bff-service service URL (get cart)
GET http://nikg77-bff-api-develop.eu-west-1.elasticbeanstalk.com/cart

For creating a new product:
POST http://nikg77-bff-api-develop.eu-west-1.elasticbeanstalk.com/products
           Content-Type: application/json
body  {"product": {"count": 100, "price": 11, "description": "TestBFF", "title": "description"}}

### CODE REVIEW

**Comment by unfalse** [comment]
What should be in .env?

**Comment by unfalse** [comment]
Product-service service API endpoint URL https://v36qxqri4i.execute-api.eu-west-1.amazonaws.com/dev
Cart-service API endpoint URL http://nikg77-cart-api-dev.eu-west-1.elasticbeanstalk.com/api/profile

That's not relevant information. You should post only endpoints for bff.

**Comment by unfalse** [comment]
How can I create a new product? I've tried a POST request to http://nikg77-bff-api-develop.eu-west-1.elasticbeanstalk.com/products
with body
```
{
    "title": "the kettle",
    "description": "a great gift for your beloved one",
    "img_url": "https://images.ru.prom.st/708352298_w640_h640_708352298.jpg",
    "price": 100
}
```

but it has returned me 400 "Invalid request body".
What is a valid request body for your service?
I should guess it myself?

**Comment by NikG77** [comment]
Sorry,  
For creating a new product:
POST http://nikg77-bff-api-develop.eu-west-1.elasticbeanstalk.com/products
Content-Type: application/json
{"product": {"count": 100, "price": 11, "description": "TestBFF", "title": "description"}}

**Comment by unfalse** [comment]
> Product-service service API endpoint URL https://v36qxqri4i.execute-api.eu-west-1.amazonaws.com/dev Cart-service API endpoint URL http://nikg77-cart-api-dev.eu-west-1.elasticbeanstalk.com/api/profile
> 
> That's not relevant information. You should post only endpoints for bff.

sorry I was wrong.

File: bff-service/app.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    const axiosConfig = {
+      method: req.method,
+      url: `${recipientUrl}${req.originalUrl}`,
+      ...(Object.keys(req.body || {}).length > 0 && { data: req.body }),
+    };
+    console.log(axiosConfig);
+
+    axios(axiosConfig)
+      .then((response) => {
+        console.log(response);
```
Comment (unfalse) [inline bff-service/app.js:29]: too much data in console.log

**Comment by unfalse** [comment]
5 points. Consise and clear implementation.

**Pull request merged**